articles/01-DownloadingTheXRTK.md

```diff
@@ -2,56 +2,63 @@
 
-Download the [XRTK-Core.unitypackage](https://github.com/XRTK/XRTK-Core/releases) asset, then drag and drop it into your project window.  This adds an Mixed Reality Toolkit seed, that will automatically register the Mixed Reality Toolkit's scoped registry with the Unity package manager, then starts searching for the latest release packages on the upm server.
+![Unity Package Manager My Registries](../images/DownloadingTheXRTK/UnityPackageManager-XRTKRegistries.jpg)
 
-![Seed Package](../images/DownloadingTheXRTK/XRTKAutoInstallAsset.png)
+Once you are ready, simply install the XRTK and any platforms you wish to use in your project to get started.
 
-The Mixed Reality Toolkit will pull all of the known packages under `com.xrtk.*` and will display them for you to easily install using the package picker window. The latest release packages are installed for you by default.
+> We recommend starting with the **XRTK.SDK** package to begin with, which will also install the Core XRTK package, for the simplest path to begin with.
 
-![Package List](../images/DownloadingTheXRTK/XRTKPackageList.png)
+## Automatic UPM Installation
+
+Our preferred deployment approach is to fully utilize [Unity's native package manager](https://docs.unity3d.com/Manual/upm-ui.html) to incorporate the Mixed Reality Toolkit in your solution, akin to the other modules Unity provides out of the box.  This is the quickest and safest way to get Mixed Reality Toolkit in your solution and is automatically updated as new releases are published.
 
-> **Note:** If you choose not to install a specific package, but choose to do so [later](#install-platform-packages).
+Download the [XRTK-Core.unitypackage](https://github.com/XRTK/XRTK-Core/releases) asset, then drag and drop it into your project window.  This adds an Mixed Reality Toolkit seed, that will automatically register the Mixed Reality Toolkit's scoped registry with the Unity package manager, then starts searching for the latest release packages on the upm server.
+
+![Seed Package](../images/DownloadingTheXRTK/XRTKAutoInstallAsset.png)
 
 ### Install platform packages
 
-To install specific platform packages you'll need to do so using the Unity Package Manager. It can be found in the editor's main menu bar:
+The Mixed Reality Toolkit will pull all of the known packages under `com.xrtk.*` and will dispaly them for you to easily install using the package picker window. The latest release pacakges are installed for you by default.
 
-```
-Window/Package Manager...
-```
+![Package List](../images/DownloadingTheXRTK/XRTKPackageList.png)
 
-![Search Package Manager](../images/DownloadingTheXRTK/PackageManager.png)
+> **Note:** If you choose not to install a specific package, but choose to do so later, you can use the Unity Package Manager's search box to filter packages by typing in `XRTK` to filter out all of the packages available in scope.
```

Comment:
I don't think this is true anymore unless you select `Custom` or `My Registries`

Reply:
Correct. Otherwise it will just serach the Unity registry. So the user first needs to switch to "My registries" or whatever it is called.

Reply:
Removed